webpack.common.config.ts

```diff
@@ -217,6 +217,11 @@ module.exports = {
     new HtmlWebpackPlugin({
       chunks: ['console_errors'],
       filename: 'console_errors.html',
+      meta: {
+        name: defaultMeta.name,
+        description: 'Contact the Oppia team, submit feedback, and learn ' +
+          'how to get involved with the Oppia project.'
```

Comment:
I don't think this is a good description for a console error page, @vojtechjelinek, is this expected?

Reply:
THe console page is only used for the e2e tests AFAIK, maybe a comment regarding this can be added.